Formula/latino.rb

```diff
@@ -0,0 +1,33 @@
+class Latino < Formula
+  desc "Lenguaje de programación de código abierto para latinos y de habla hispana"
+  homepage "https://www.lenguajelatino.org/"
+  license "MIT"
+  stable do
+    url "https://github.com/lenguaje-latino/Latino/archive/v1.3.0.tar.gz"
+    sha256 "28118a047265394ad9a5fcb2c76895d08af4b43a5886f52afce98a0f8cdc86fe"
+    depends_on "cmake" => :build
+    depends_on "make" => :build
+  end
+  head do
+    url "https://github.com/lenguaje-latino/Latino.git"
+    depends_on "cmake" => :build
+    depends_on "make" => :build
+  end
+  fails_with :clang do
+    build 600
+    cause "multiple configure and compile errors"
+  end
+  fails_with :gcc do
+    build 600
+    cause "multiple configure and compile errors"
+  end
```

Comment:
```suggestion
  url "https://github.com/lenguaje-latino/Latino/archive/v1.3.0.tar.gz"
  sha256 "28118a047265394ad9a5fcb2c76895d08af4b43a5886f52afce98a0f8cdc86fe"
  license "MIT"
  head "https://github.com/lenguaje-latino/Latino.git"
  
  depends_on "cmake" => :build
  depends_on "make" => :build
```

Will this build with the built-in macOS `make`? If it does, then you don't need to declare `depends_on "make"`.

I think we can probably also skip the `fails_with` blocks.

Reply:
1. yes this will build with the built-in macOS make. 

2. okay will get rid fo depends on make 

3. okay will get rid of fails_with

Reply:
Great. You also don't need the `stable` and `head` blocks; see the suggestion in my previous comment.